.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,8 +15,8 @@
 ## Links to Tickets or other pull requests
 <!--
 Base links to copy
-- https://github.com/schul-cloud/schulcloud-server/pull/????
-- https://ticketsystem.schul-cloud.org/browse/SC-????
+- https://github.com/hpi-schul-cloud/schulcloud-client/pull/????
```

Comment:
Is there a reason why it change vom server to client?

Reply:
Yes this is the template to link the related client prs.

Reply:
okay that is fine then :)